packages/@aws-cdk/region-info/lib/region-info.ts

```diff
@@ -111,10 +111,17 @@ export class RegionInfo {
   }
 
   /**
-   * The ID of the AWS account that owns the public ECR repository that contains the
-   * AWS App Mesh Envoy Proxy images in a given region.
+   * The ID of the AWS account that owns the public ECR repository containing the
+   * AWS App Mesh Envoy images in this region.
    */
   public get appMeshRepositoryAccount(): string | undefined {
-    return Fact.find(this.name, FactName.APPMESH_ECR_ACCOUNT);
+    return Fact.find(this.name, FactName.APPMESH_REPOSITORY_ACCOUNT);
+  }
+
+  /**
+   * The image name and tag of the latest Envoy release from AWS App Mesh.
+   */
+  public get appMeshLatestEnvoyImage(): string | undefined {
```

Comment:
So I have a problem with the `latest` part of the name... This will only ever be the "latest" version at time of publishing the current `@aws-cdk/region-info` release (and even then - this assumes someone diligently updates the facts table every time the value changes).

If we want to have this, we need to come up with a name that does *not* include `latest`, as this is a false promise.